### ISSUE TITLE
configuration for devise

### ISSUE THREAD

**Issue by vmvwebworks** [omniauth/omniauth-saml#129]
After configure, i tried to start the server and said this:

Wrong OmniAuth configuration. If you are getting this exception, it means that either:

1) You are manually setting OmniAuth.config.path_prefix and it doesn't match the Devise one
2) You are setting :omniauthable in more than one model
3) You changed your Devise routes/OmniAuth setting and haven't restarted your server my-profile-configuration/gems/devise-3.2.4/lib/devise/rails/routes.rb:471:in `set_omniauth_path_prefix!

I have this on my user model

devise :registerable, :confirmable, :recoverable, :rememberable, :trackable, :validatable, :database_authenticatable, :omniauthable, :omniauth_providers => [:saml]

I follow the instructions include when it says to follow omniauth facebook example and got this. What would be done?

**Comment by ilikepi** [comment]
Hello,

Could you please also post the `devise_for` statement from your `config/routes.rb` file?

**Comment by vmvwebworks** [comment]
Of course

devise :registerable, :confirmable, :recoverable,
         :rememberable, :trackable, :validatable, :database_authenticatable, :omniauthable, :omniauth_providers => [:saml]

**Comment by vmvwebworks** [comment]
http://stackoverflow.com/questions/24574094/deviseomniauth-routes-versioning

The solution that uncomment the line at the bottom of the file, it says:
`# config.omniauth_path_prefix = '/my_engine/users/auth'`
and add your route

**Comment by ilikepi** [comment]
Hi @vmvwebworks, 

Can you please confirm that the solution in that Stack Overflow post resolved your issue?  Or are you still encountering the error?

**Comment by vmvwebworks** [comment]
Yes, it solved, at least for now. I still developing the same feature, i will notify here if i found more issues.

**Comment by dwoznicka** [comment]
thanks for this @vmvwebworks , i've encountered same issue and couldn't find any solution until now. would be best if we could add this to readme

**Comment by Romain-C** [comment]
The fix is nice but as the config file is global than it doesn't work when using multiple devise model and not all are on the namespace. Any ideas on how to achieve this path fix for only one devise model/route ?

**Comment by motazbalghonaim** [comment]
We are currently running into this issue too. Any updates or potential workarounds?
